common/persistence/persistenceTestBase.go

```diff
@@ -168,10 +168,13 @@ func (s *TestBase) SetupWorkflowStoreWithOptions(options TestBaseOptions) {
 	s.readLevel = 0
 	s.replicationReadLevel = 0
 	s.ShardInfo = &ShardInfo{
-		ShardID:             shardID,
-		RangeID:             0,
-		TransferAckLevel:    0,
-		ReplicationAckLevel: 0,
+		ShardID:                 shardID,
+		RangeID:                 0,
+		TransferAckLevel:        0,
+		ReplicationAckLevel:     0,
+		TimerAckLevel:           time.Time{},
```

Comment:
why do we need a separate TimerAckLevel.  Can we use the cluster map for current cluster too?

Reply:
we can, however, when loading a ack level for a new cluster, say "cluster foo", we will use the 
ClusterTimerAckLeve["current cluster"] to initialize which seems to be weird.